test/benchmarks.cc

```diff
@@ -0,0 +1,44 @@
+#include <catch2/catch.hpp>
+
+#include <intrusive_graph.h>
```

Comment:
We use quotes (`"intrusive_graph.h"`) for our own headers by convention everywhere else. We should align this.

Reply:
I actually checked in `intrusive_graph_tests.cc` which is the same type of file in the same folder with the same use case, and it uses `<>`, that's why I went with that. I can change both I guess.

Reply:
Done.